feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts

```diff
@@ -352,7 +473,9 @@ export class ConfiguratorAddToCartButtonComponent implements OnInit, OnDestroy {
         .pipe(
           take(1),
           delay(0),
-          map(() => this.configUtils.getElement('.cx-price-summary-container')),
+          map(() =>
```

Comment:
Why do we have this change?

Reply:
For display only mode, for instance in order view, there is no price summary